docs/pages/blog/mui-x-sep-2024-price-update.md

```diff
@@ -2,57 +2,74 @@
 title: Upcoming changes to MUI X pricing in 2024
 description: Check out the new pricing updates and how to transition to the new model.
 date: 2024-08-01T08:00:00.000Z
-authors: ['josefreitas']
+authors: ['josefreitas', 'oliviertassinari']
 tags: ['MUI X', 'Product']
 manualCard: true
 ---
 
-We're excited to share some important updates to our product offerings and pricing structure. This change reflects the expanded feature set and helps us continue delivering high-quality components to you and your users.
+Today, we are sharing some important updates to the MUI X product offerings and pricing structure.
+These changes are made to reflect the expanded feature set and help support the delivering of high-quality components to you and your users.
 
 ## Changes to the Pro plan
 
-### Current Pricing
+### Current pricing
 
-Under our current pricing model, if you licensed the Pro plan for 50 developers, you only paid for 10 seats. This concept was introduced when we offered only the Data Grid Pro with a much smaller feature set.
```

Comment:
Removed the notion of "seat", I believe we banned it in the legal an pricing page.

Reply:
Hmm, okay. I guess I missed that discussion.

Reply:
Fixing that too in #14351.

Reply:
Why though? Shouldn't we have separate terms for the license (permission to use the software) and the number of devs that a single license covers?

Reply:
I agree but https://github.com/mui/material-ui/pull/43438#discussion_r1731566149